routing-daemon/lib/openshift/routing/models/f5-icontrol-rest.rb

```diff
@@ -249,13 +268,11 @@ def initialize logger, cfgfile
 
       read_config cfgfile
 
-      # Create the openshift_application_routes an openshift_application_aliases
-      # policies if they do not already exist.  The create_route and
-      # delete_route methods add and delete the application-specific rules to
-      # and from the former, and the add_pool_alias and delete_pool_alias
+      # Create the openshift_application_aliases
+      # policy if they do not already exist. The add_pool_alias and delete_pool_alias
       # methods add and delete the application-specific rules to and from the
       # latter.
-      ["openshift_application_routes","openshift_application_aliases"].each do |policy|
+      ["openshift_application_aliases"].each do |policy|
```

Comment:
The `each` is redundant now; delete it and the `do |policy|`/`end`, rewrap, and replace `policy` with `"openshift_application_aliases"`.